velox/vector/fuzzer/VectorFuzzer.cpp

```diff
@@ -728,13 +726,15 @@ RowVectorPtr VectorFuzzer::fuzzRow(const RowTypePtr& rowType) {
 
 RowVectorPtr VectorFuzzer::fuzzRow(
     const RowTypePtr& rowType,
-    vector_size_t size) {
+    vector_size_t size,
+    bool allowTopLevelNulls) {
```

Comment:
nit: why not just use containerNulls ? and remove the option that forcefully sets it to false ?